src/overlay/test/LoopbackPeer.cpp

```diff
@@ -273,13 +273,22 @@ LoopbackPeer::recvMessage(xdr::msg_ptr const& msg)
     catch (xdr::xdr_runtime_error& e)
     {
         CLOG_ERROR(Overlay, "received corrupt xdr::msg_ptr {}", e.what());
-        drop("received corrupted message",
-             Peer::DropDirection::WE_DROPPED_REMOTE,
-             Peer::DropMode::IGNORE_WRITE_QUEUE);
+        drop("git received corrupted message",
```

Comment:
I think the `git` in this line is unintentional?

Reply:
oops, typo. fixed.